doc/doxygen/code-gallery/CMakeLists.txt

```diff
@@ -21,6 +21,9 @@
 
 ADD_CUSTOM_TARGET(code-gallery)
 
+MESSAGE(STATUS "Setting up code gallery documentation from ${DEAL_II_CODE_GALLERY_DIRECTORY}.")
```

Comment:
On second thought, I would move it back into the IF. Any good reason you moved it outside?